### PR TITLE
chore(octon): complete wave 5 agency and adapter hardening

### PR DESCRIPTION
**Pull request by jamesryancooper**
Policy reference: `.octon/framework/agency/practices/pull-request-standards.md`
Reminder: Run `@codex review`.

## What

Completes Wave 5 across the live Octon harness by simplifying the agency kernel, hardening adapter contracts, bounding support-target declarations, and aligning disclosure/runtime evidence with the new model.

## Why

Wave 5 required one accountable orchestrator as the default execution role, replaceable non-authoritative host/model adapters, and no hidden-authority drift across runtime, docs, validators, or disclosure artifacts. No-Issue: repository-governance Wave 5 completion work.

## How

Replaced persona-heavy `architect` / `auditor` defaults with `orchestrator` / `verifier`, published constitutional adapter contracts plus runtime adapter manifests, extended support-target declarations and runtime routing to fail closed on undeclared adapter envelopes, and updated disclosure contracts so RunCards and HarnessCards carry adapter provenance and conformance criteria. I also normalized remaining active role-ID drift on live orchestration/service/example surfaces and refreshed the generated/effective publication outputs produced by the final validation sweep.

## Profile Selection Receipt

- Semantic version source(s): `/.octon/octon.yml`
- Release state (`pre-1.0` or `stable`): `pre-1.0`
- `change_profile` (`atomic` or `transitional`): `transitional`
- Hard-gate facts (downtime, coordination, migration/backfill, rollback, blast radius, compliance): runtime support routing, agency discovery, constitutional registries, and validators had to land together; adapter support declarations had to exist before fail-closed runtime enforcement; rollback requires reverting the full Wave 5 set together.
- Tie-break status: `transitional` selected because runtime, agency, disclosure, and validator cutovers needed to coexist until the full branch landed.
- `transitional_exception_note` (required when `pre-1.0` + `transitional`): simplify the agency kernel and publish adapter contracts without leaving a window where runtime, docs, and validators disagree on authority; owner `Octon governance`; target removal `2026-06-30`.

## Implementation Plan

- Workstreams and scope:
  - simplify `framework/agency/**` around `orchestrator` and `verifier`
  - publish `framework/constitution/contracts/adapters/**` and `framework/engine/runtime/adapters/**`
  - extend `instance/governance/support-targets.yml` and runtime routing for adapter envelopes/conformance
  - align disclosure contracts, retained sample cards, bootstrap docs, scaffolding, and validators
- Public interfaces/types/contracts affected:
  - agency registries and role contracts
  - constitutional adapter contract family
  - support-target schema/declaration
  - disclosure schemas for RunCard and HarnessCard
  - engine policy-interface and runtime authorization metadata
- Test scenarios:
  - full `alignment-check --profile harness,agency`
  - Wave 5 validator stack
  - disclosure/runtime governance validators
  - targeted orchestration shell tests
  - targeted `octon_kernel`, `octon_core`, and `octon_studio` tests
- Assumptions/defaults:
  - `repo-shell` is the default host adapter
  - `repo-local-governed` is the default model adapter
  - generated/effective publication receipts created by the validation sweep are part of the branch state

## Impact Map (code, tests, docs, contracts)

- Code:
  - `/.octon/framework/engine/runtime/crates/{kernel,core,studio}/**`
  - `/.octon/framework/orchestration/runtime/_ops/scripts/write-run.sh`
  - `/.octon/framework/lab/runtime/_ops/scripts/write-harness-card.sh`
- Tests:
  - `/.octon/framework/orchestration/runtime/_ops/tests/**`
  - `/.octon/framework/assurance/runtime/_ops/tests/**`
- Docs:
  - `/.octon/framework/agency/**`
  - `/.octon/framework/cognition/_meta/architecture/**`
  - `/.octon/instance/bootstrap/**`
  - `/.octon/instance/cognition/context/shared/migrations/2026-03-27-wave5-agency-simplification-adapter-hardening-cutover/plan.md`
  - `/.octon/instance/cognition/decisions/073-wave5-agency-simplification-and-adapter-hardening-cutover.md`
- Contracts:
  - `/.octon/framework/constitution/contracts/{adapters,disclosure}/**`
  - `/.octon/framework/constitution/{CHARTER.md,charter.yml,precedence,obligations,support-targets.schema.json}`
  - `/.octon/instance/governance/support-targets.yml`
  - `/.octon/octon.yml`

## Compliance Receipt

- [x] Selected exactly one execution profile before planning/implementation.
- [x] Applied release-maturity gate from semantic version.
- [x] Enforced pre-1.0 default (`atomic`) unless hard gates required `transitional`.
- [x] Included `transitional_exception_note` when required.
- [x] Included tie-break escalation when applicable.
- [x] Updated impacted contracts/docs/tests.
- [x] Honored charter change-control constraint (no direct principles charter edits without override evidence).

## Exceptions/Escalations

none

## Tradeoffs

The branch is large because Wave 5 required coordinated agency, constitution, runtime, disclosure, generated/effective publication, and evidence updates in one change set. Some historical or reference-only uses of words like “architect” remain outside active runtime/governance authority because they are not live Octon role IDs.

## Testing

- `bash .octon/framework/assurance/runtime/_ops/scripts/alignment-check.sh --profile harness,agency`
- `bash .octon/framework/agency/_ops/scripts/validate/validate-agency.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-wave5-agency-adapter-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-assurance-disclosure-expansion.sh`
- `bash .octon/framework/assurance/runtime/_ops/scripts/validate-execution-governance.sh`
- `bash .octon/framework/orchestration/runtime/_ops/tests/test-incident-approval-control.sh`
- `bash .octon/framework/orchestration/runtime/_ops/tests/test-operator-hardening.sh`
- `bash .octon/framework/assurance/runtime/_ops/tests/test-validate-continuity-memory.sh`
- `cargo test --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_kernel authorization::tests::undeclared_host_adapter_denies_execution -- --exact`
- `cargo test --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_kernel workflow::tests::create_design_package_writes_execution_artifacts -- --exact`
- `cargo test --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_core orchestration::tests::incident_closure_readiness_reports_missing_evidence -- --exact`
- `cargo test --manifest-path .octon/framework/engine/runtime/crates/Cargo.toml -p octon_studio app_state::tests::operations_incident_blockers_and_playbooks_render -- --exact`

## Rollout

Draft PR only. Merge after required checks are green and the explicit `accept:human` acknowledgement remains present for the high-impact governance/runtime changes.

## Convivial Purpose Check

- [x] Feature expands genuine user capability (not synthetic engagement).
- [x] Attention and interruption behavior are justified and user-controllable.
- [x] No manipulative patterns or dark-pattern mechanics are introduced.
- [x] Data collection/extraction risk is minimal and explicitly justified.

## Risk Rubric

- Risk class: [ ] Trivial [ ] Low [ ] Medium [x] High
- Rollback plan: revert the full Wave 5 change set together
- Rollback handle: `git revert 25a97905c`
- Flags changed (name, owner, expiry, rollout): none
- Autonomy eligibility: [ ] autonomy:auto-merge [x] autonomy:no-automerge

## Contracts and Threat Model

- OpenAPI/JSON-Schema changes:
  - adapter contract schemas
  - support-target schema
  - RunCard and HarnessCard disclosure schemas
- Threat model update/link:
  - no hidden-authority widening through host/model adapters
  - support claims remain bounded by support-target declarations and conformance criteria

## Observability and Performance

- Traces/logs/metrics for changed flows:
  - runtime/disclosure evidence now includes adapter provenance in retained sample cards
  - generated/effective publication receipts refreshed during validation
- Representative traces for high risk changes:
  - runtime-effective-state and execution-governance validation receipts in the Wave 5 evidence bundle
- Performance or bundle impact:
  - negligible runtime cost; mostly governance/validation/disclosure metadata expansion

## License and Provenance

- New dependencies and licenses:
  - none
- Generated code/templates provenance notes:
  - generated/effective publication outputs and publication receipts were refreshed by the local validation/publish scripts in this branch

## Checklist

- [x] Requirements met; edge cases handled
- [x] Security reviewed (authz, input validation, secrets)
- [x] Tests added or updated
- [x] Observability updated (logs, metrics, traces) if needed
- [x] No speculative abstractions or unnecessary complexity
- [x] Conventions followed; no drift introduced
- [x] Non-obvious decisions documented (comments, ADR)
- [x] All review conversations resolved

## Screenshots / Notes

- Final comprehensive sweep found one remaining integration miss: adapter contract README authoritative-doc classification. That was corrected before the final clean pass.
- Wave 5 completion is recorded in:
  - `/.octon/instance/cognition/context/shared/migrations/2026-03-27-wave5-agency-simplification-adapter-hardening-cutover/plan.md`
  - `/.octon/instance/cognition/decisions/073-wave5-agency-simplification-and-adapter-hardening-cutover.md`
  - `/.octon/state/evidence/migration/2026-03-27-wave5-agency-simplification-adapter-hardening-cutover/validation.md`

### CODE REVIEW

File: .octon/framework/orchestration/runtime/_ops/scripts/write-run.sh [hunk 1]
```diff
@@ -54,6 +54,28 @@ validate_support_tier() {
   exit 1
 }
 
+adapter_status() {
+  local adapter_kind="$1"
+  local adapter_id="$2"
+  local support_targets_file="$OCTON_DIR/instance/governance/support-targets.yml"
+  local query=".$adapter_kind[] | select(.adapter_id == \"$adapter_id\") | .support_status // \"unsupported\""
+  yq -r "$query" "$support_targets_file" 2>/dev/null | head -n 1
```
[the remark this answers is not in the document]
Comment (jamesryancooper) [reply .octon/framework/orchestration/runtime/_ops/scripts/write-run.sh]: Addressed in `e7043e457` (`fix(runtime): preserve unsupported adapter disclosure state`). `adapter_status` now falls back to `unsupported` when a lookup yields no rows, so undeclared host/model adapters no longer emit empty enum values into RunCard disclosure. I revalidated this path with `bash -n .octon/framework/orchestration/runtime/_ops/scripts/write-run.sh`, `bash .octon/framework/assurance/runtime/_ops/scripts/validate-assurance-disclosure-expansion.sh`, and `bash .octon/framework/assurance/runtime/_ops/scripts/validate-runtime-effective-state.sh`.
